test/e2e/sched/sched_test.go

```diff
@@ -264,10 +272,300 @@ var _ = Describe("[Scheduler] CR configuration management", func() {
 			dp, err := podlist.With(e2eclient.Client).DeploymentByOwnerReference(context.TODO(), nroSchedObj.UID)
 			Expect(err).ToNot(HaveOccurred(), "unable to get deployment by owner reference")
 
+			klog.Info("checking the old pod is removed")
+			err = wait.With(e2eclient.Client).Timeout(3*time.Minute).ForPodDeleted(context.TODO(), podList[0].Namespace, podList[0].Name)
+			Expect(err).ToNot(HaveOccurred())
+
 			podList, err = podlist.With(e2eclient.Client).ByDeployment(context.TODO(), *dp)
 			Expect(err).NotTo(HaveOccurred())
 			Expect(podList).To(HaveLen(1), "cannot find correct amount of pods for DP %s/%s", dp.Namespace, dp.Name)
 			Expect(podList[0].UID).ToNot(Equal(uid), "new scheduler pod has not been created")
 		})
+
+		It("should be able to modify scheduler loglevel", func() {
+			deployment, err := podlist.With(e2eclient.Client).DeploymentByOwnerReference(context.TODO(), initialSchedObj.UID)
+			Expect(err).ToNot(HaveOccurred())
+			initialPodList, err := podlist.With(e2eclient.Client).ByDeployment(context.TODO(), *deployment)
+			Expect(err).NotTo(HaveOccurred())
+			for i, pod := range initialPodList {
+				fmt.Printf("pod %d by sched deployment: %s/%s", i, pod.Namespace, pod.Name)
+			}
+
+			Expect(initialPodList).To(HaveLen(1), "cannot find correct amount of pods for DP %s/%s", deployment.Namespace, deployment.Name)
```

Comment:
We want to set replica=len(controlPlane) by default, so let's just check we have at least 1 element